### PR TITLE
Add some more useful development tools

### PR DESCRIPTION
**Pull request by conatus**
Add some more useful development tooling to the mix.

### [Create Block Theme](https://wordpress.org/plugins/create-block-theme/)

Save off block themes that you are making in the full site editor to the theme on disk. Add fonts direct from the WordPress backend.

### [Pattern Manager](https://wordpress.org/plugins/pattern-manager/)

Create WordPress patterns from within the WordPress backend. They are automatically saved to the theme on disk, with appropriate metadata and can also be modified in this way.

### [Fakerpress](https://wordpress.org/plugins/fakerpress/)

Generate a lot of dummy content to see how the website looks and feels with full content in it.

### [Yoast Duplicate Posts](https://wordpress.org/plugins/duplicate-post/)

Again for creating dummy data, this allows quick duplication of posts to fill things out.

### CODE REVIEW

**Comment by conatus** [comment]
FYI all other developers.

**Comment by conatus** [comment]
Will wait a day for feedback then merge.

File: README.md [hunk 1]
```diff
@@ -88,4 +108,4 @@ These are exhaustive manual instructions, but should not be required after initi
 16. Still on the tools page, setup Force HTTPS by clicking on Modify and dollowing your nose, selecting "Force all traffic to the primary domain" along the way. Edit the `.env` file created in step 12, to have `WP_HOME` include `https` not `http`.
 17. In Domains, change the DNS pointed domain to your primary one selecting "Make primary".
 18. WordPress should now work at the site URL. You can login to the administration dashboard with the password you just created.
-19. Repeat steps 7 through 20, but creating a development environment.
+19. Repeat steps 7 through 20, but creating a developmgent environment.
```
Comment (janbaykara) [inline README.md:111]: woopsie

**Review by janbaykara — CHANGES_REQUESTED** [review]
Appreciate the links and docs and so on! Just a typo to fix.

**Comment by conatus** [comment]
@janbaykara 

Boink.

**Review by janbaykara — APPROVED** [review]
LGTM!

**Pull request merged**